### PR TITLE
Feature/test news stories

### LINKED ISSUE #53: Create tests for NewsStories component
- [x] display the title
- [x] display the footer
- [x] display news cards
- [x] display buttons inside news cards
- [x] click take action button
- [x] display modal
- [x] click the title in the modal and redirect to CongressAction component

### PR DESCRIPTION
**Pull request by camianderson**
Hi Team, Please review this PR. 

## Description of Changes:
* Create tests for NewsStories component
* Stub the API call for testing purposes

## Put an 'X' next to all that apply
New Test: [x]
New Feature:[]
Bug Fix: []

## Before You Submit Ensure
All Tests Pass: [x]
Runs Locally: [x]

Closes #53 

Thanks! 

-Cami

### CODE REVIEW

**Review by Neil-B-Linton — APPROVED** [review]
looks good to me

File: cypress/e2e/NewsStories.cy.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        cy.url().should('eq', 'http://localhost:3000/');
+    })
+
+    it('Should be able to see the footer', () => {
+        cy.get('.footer').should('contain', 'Hotline');
+    })
+
+    it('Should be able to click the Take Action button and see the CongressAction option', () => {
+        cy.selectTopic();
+        cy.intercept('POST', 'https://3bb63f73-e6d2-4120-8a53-4f012668a561.mock.pstmn.io/graphql?test=1', (req) => {
```
Comment (jordan-sullivan) [inline cypress/e2e/NewsStories.cy.js:56]: Amazing job stubbing and intercepting!

File: cypress/e2e/NewsStories.cy.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            const { body } = req
+            aliasQuery(req, 'News')
+            if(hasOperationName(req, 'News')){
```
Comment (jordan-sullivan) [inline cypress/e2e/NewsStories.cy.js:59]: Fantastic logic here!

File: cypress/e2e/NewsStories.cy.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        });
+        cy.get('.btn').eq(0).click();
+        cy.get('.mpo-modal-body').should('exist');
+        cy.contains('.congress-action', 'Write to your local Congress member');
+        cy.contains('p', 'Take action by writing a letter or making a call to your local congress person about this issue!');
+    })
+
+    it('Should be able to click the CongressAction option inside the modal', () => {
+        cy.selectTopic();
+        cy.intercept('POST', 'https://3bb63f73-e6d2-4120-8a53-4f012668a561.mock.pstmn.io/graphql?test=1', (req) => {
```
Comment (jordan-sullivan) [inline cypress/e2e/NewsStories.cy.js:88]: nice!

**Review by jordan-sullivan — APPROVED** [review]
Great work getting the stubbs to work!

**Pull request merged**